config/vudl.yml

```diff
@@ -14,6 +14,12 @@ defaults: &defaults
   # Settings for ocrmypdf (omit setting to skip this step if you aren't using it)
   #ocrmypdf_path: /usr/local/bin/ocrmypdf
 
+  #VuFind Base URL
+  vufind_base_url: https://myserver
```

Comment:
It looks like the code only uses vufind_test_url; what is the purpose of having separate vufind_base_url and vufind_test_url settings? (I'd lean toward just using vufind_base_url everywhere).